ams/dashboard/vite.config.ts

```diff
@@ -0,0 +1,62 @@
+import { defineConfig } from 'vite'
+import vue from '@vitejs/plugin-vue'
+import { viteMockServe } from 'vite-plugin-mock'
+import path from 'node:path'
+
+const css = {
+  preprocessorOptions: {
+    less: {
+      modifyVars: {
+        'primary-color': '#1890ff',
+        'link-color': '#1890ff',
+        'border-color-base': '#e8e8f0',
+        'border-radius-base': '2px',
+        'border-color-split': '#e8e8f0',
+        'header-color': 'rgba(0, 0, 0, 0.85)',
+        'text-color': '#79809a',
+        'text-color-secondary': '#c0c0ca',
+        'font-size-base': '14px',
+        'dark-gray-color': '#2b354a',
+        'dark-bg-color': '#202a40',
+        'dark-bg-primary-color': '#1a2232'
+      },
+      javascriptEnabled: true
+    }
+  }
+}
+
+// https://vitejs.dev/config/
+export default defineConfig({
```

Comment:
We need to build the static resources into the `/src/main/resources/static/ `directory, and we need to commit the packaged static resources, so we may need to configure the `build.outDir`